src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -384,7 +387,7 @@ private IEnumerable<string> CollectInputs(BuildUpToDateCheckLogger logger)
             if (_customInputs.Count != 0)
             {
                 logger.Verbose("Adding " + UpToDateCheckInput.SchemaName + " inputs:");
-                foreach (string input in _customInputs.Select(_configuredProject.UnconfiguredProject.MakeRooted))
+                foreach (string input in _customInputs.Select(s => _configuredProject.UnconfiguredProject.MakeRooted(s)))
```

Comment:
This looks like another compiler bug, but I wasn't sure how to file it.

Reply:
This actually gives better performance anyway as the compiler caches the delegate for lambdas but not for method groups.

Reply:
I thought that changed very recently, but I could be wrong.

Reply:
Nice! I can't reproduce the old behaviour any more, even on 7.3:

https://sharplab.io/#v2:EYLgZgpghgLgrgJwgZwLQFEAeAHJzkCWA9gHYBqUCBUwANigD4ACADAARMCMA3ALABQTAMwcATGwDCbAN4C28jiKYAWNgFkAFAEo2chbP4KjbAIIa1WvoePyz2tgF4AfOu2W9Nj8YBiGgHLu1ja+9s5sfm5WNvJeRgBulGwAJo5sJBAA7mwAItpR0b5JAHQW+cEaoS7FmlqBRgC+Xl64BAkwEGwEJDDh9tIcAOxsnKJC3GyNQbpTwhyqZlziUDrSk0azKmzeADwAKk4aTACsey5gK2sKXkzi2TKxCrNdPTUyTEMjY5fyk/VAA===